boards/st/nucleo_g431kb/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: nucleo_g431kb
```

Comment:
```suggestion
  name: nucleo_g431kb
  full_name: Nucleo G431KB
```

Although in fact I think the official name should be spelled NUCLEO-G431KB. Wonder if it's worth a global fix for all other Nucleo boards? @erwango

Reply:
Indeed, if the goal is to stick exactly with ST reference, actual name is all caps.

Reply:
We can work on a follow up PR to fix this